front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -195,6 +195,7 @@ class Sidebar extends PureComponent<
           iconName: 'sidebar-proposals',
           message: 'initiatives',
           featureNames: ['initiatives'],
+          onlyCheckAllowed: true,
```

Comment:
To make sure the admin sidebar item doesn't disappear when toggling off the feature.